pkg/interceptors/recovery.go

```diff
@@ -32,7 +31,6 @@ func RecoveryStreamServerInterceptor() grpc.StreamServerInterceptor {
 var recoveryOption = []grpcrecovery.Option{
 	grpcrecovery.WithRecoveryHandlerContext(func(ctx context.Context, rec interface{}) (err error) {
 		sentry.CurrentHub().Recover(rec)
-		sentry.Flush(time.Second * 5)
```

Comment:
[q]: shouldn't we remove `sentry.CurrentHub().Recover(rec)` as well?

Reply:
@Neurostep I believe you mean remove it here and append it to the sentry hook, right?
If so, could you provide a code sample so we can compare the options?

Reply:
@laynax 

>I believe you mean remove it here and append it to the sentry hook, right?

No, I mean delete the statement as we don't need it. Essentially, all it does is call the [CaptureEvent](https://github.com/getsentry/sentry-go/blob/77f95170e7f35f5c2c16bd353bed0e99a3696c82/client.go#L498) function which we do in the hook already.

Reply:
@Neurostep 
We are using fatal without error `l.Fatalf("http: panic serving URI %s: %v", r.URL.RequestURI(), rec)` here so it's not captured. The reason for that is because the recovered object is an `interface{}` and `WithError()` accepts error.
I believe `sentry.CurrentHub().Recover(rec)` is used properly here.

Reply:
Moving the `Recover` to the logger package as well 👍